dev/src/Command/AddComponentCommand.php

```diff
@@ -111,17 +123,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->render();
         }
 
-        $productHomepage = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product homepage? ')
-        );
-
-        $productDocumentation = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product documentation URL? ')
-        );
+        $version = basename(dirname($proto));
+        // yaml file contains documentation_uri and
+        // it is available at the same location as proto file
+        $libraryName = strtolower($new->componentName);
+        $yamlFileName = $libraryName . '_' . $version . '.yaml';
+        $yamlFilePath = dirname($proto) . '/' . $yamlFileName;
+        $googleApisDir = explode('/google/', $yamlFilePath)[0];
```

Comment:
I think this is the same as `$new->protoPath`

Reply:
Its different actually!
![Uploading Screenshot 2023-12-12 at 19.32.55.png…]()

Reply:
The screenshot did not upload

Reply:
<img width="430" alt="Screenshot 2023-12-12 at 19 32 55" src="https://github.com/googleapis/google-cloud-php/assets/14087896/96bdf2fc-4d14-4bd0-a917-d5a5925913e6">

Reply:
Ahh I see, `protoPath` is not the right one. But you can use the `$protoFile` variable that exists earlier in the function, and then pass that to `loadProtoContents` to get the value you want:

```php
$protoDir = dirname($protoFile);
$yamlContents = $this->loadProtoContent($protoDir . '/' . $yamlFilename);
```

Reply:
Yeah, this is already implemented in the code.

Reply:
here: https://github.com/googleapis/google-cloud-php/pull/6861/files#diff-32c7b6783670f3e2cab4dd8bce5eae4ee32cfd32d4ca78b560ce6ebbcd080c7aR129